protocol/messenger_handler.go

```diff
@@ -1622,11 +1622,20 @@ func (m *Messenger) HandleCommunityRequestToJoinResponse(state *ReceivedMessageS
 			communitySettings := response.CommunitiesSettings()[0]
 			community := response.Communities()[0]
 			state.Response.AddCommunity(community)
+			chatIDs, err := m.getChatIdsForCommunity(community.ID())
+			if err != nil {
+				return err
+			}
+
+			if len(chatIDs) > 0 {
+				generalChannelChatID := m.Chat(chatIDs[0])
```

Comment:
The admin can remove the `#general` channel after creation.

Reply:
@siddarthkay maybe put the pr back to draft if it's not ready, we dicussed a different way to handle this, we should merge messenger responses